tests/rados/test_mon_pg_warn.py

```diff
@@ -105,7 +104,7 @@ def mon_pg_warn_health_warn(should_contain: bool):
                         pg_count = pool_obj.get_target_pg_num_bulk_flag(
                             pool_name=pool_name
                         )
-                        assert pg_count > 1
+                        assert pg_count >= 32
```

Comment:
It can be equal to 32 if the cluster already has multiple pools... I feel better to remove the check for equal.

Reply:
It's only checking for the current pool which was created during this test case execution